auslib/test/test_db.py

```diff
@@ -3816,7 +3817,7 @@ def doit():
     def testOnInsertRuleSC(self):
         def doit():
             self.db.rules.scheduled_changes.insert("bob", when=2000000000000000, product="foo", channel="bar", backgroundRate=100, priority=50,
-                                                   update_type="minor", change_type="insert")
+                                                   update_type="minor", change_type="insert", base_rule_id=1)
```

Comment:
Is this a necessary change? This should work without specifying rule_id, because it is autoincrement. I think base_rule_id ends up getting ignored, maybe, because insert wants columns specified without base_.